src/components/SummaryTable.test.js

```diff
@@ -130,7 +132,14 @@ const testOptionalAnswers = {
   establishment_secondary_number: "7654321",
   operator_secondary_number: "7654321",
   partners: ["John", "Doe"],
-  main_partnership_contact: "John"
+  main_partnership_contact: "John",
+  opening_hours_monday: "9:30 - 19:00",
+  opening_hours_tuesday: "09:30 - 19:00",
+  opening_hours_wednesday: "9:30am - 7pm",
+  opening_hours_thursday: "0930 - 1900",
+  opening_hours_friday: "9:30 to 19:00",
+  opening_hours_saturday: "09:30 to 19:00",
+  opening_hours_sunday: "From 9:30 to 19:00"
```

Comment:
Is it just me or are these values never tested? I know you're just fitting in with existing method of test.

Reply:
This is only to test row counts